src/Development/IDE/GHC/Orphans.hs

```diff
@@ -10,14 +10,14 @@
 --   Note that the 'NFData' instances may not be law abiding.
 module Development.IDE.GHC.Orphans() where
 
-import GHC
-import GhcPlugins
-import Development.IDE.GHC.Compat
-import qualified StringBuffer as SB
-import Control.DeepSeq
-import Data.Hashable
-import Development.IDE.GHC.Util
-import Bag
+import           Bag
```

Comment:
As a project we don't like this style - better not to indent things more than necessary.

Reply:
I seem to have made a mistake, the first two commits have already been accepted as part of a previous PR.

The one I meant to submit is only the last one (exposing Development.IDE.Core.Preprocessor).

Reply:
Sorry for the confusion, I have opened a different PR with just the required change.